srfi-119/srfi-testsuite.html

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+<!--
+SPDX-FileCopyrightText: 2012 - 2013 Free Software Foundation, Inc.
+SPDX-FileCopyrightText: 2014 Arne Babenhauserheide <arne_bab@web.de>
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
```

Comment:
This file is mixed-content: the Javascript code is GPL (reused from org-mode), but the tests themselves are MIT to enable for seamless re-use in Schemes implementing the SRFI.

See https://hg.sr.ht/~arnebab/wisp/browse/README.md?rev=tip (section License).

Reply:
Thanks, fixed.  I've also left the MIT license text in place as I believe that's the current consensus that emerged from discussions. The GPL notice can be replaced, according to the same discussion.

Reply:
This is the testsuite which does not have to use a lax license (as by the original discussions back when the SRFI was created), because it is not part of the actual implementation.

Reply:
OK, will remove the MIT from the test suite then, if that's your preference.